Description: Making Twisted work from file:// PyPI URI.
 File paths are case sensitive, and we use file:// for PyPI deps in
 debian package builds.

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- conn-check-1.0.13.orig/requirements.txt
+++ conn-check-1.0.13/requirements.txt
@@ -4,6 +4,6 @@ pyasn1
 pyOpenSSL
 service_identity
 requests >= 2.0
-twisted
+Twisted
 txrequests >= 0.9
 pyyaml
